doc/faq.rst

```diff
@@ -330,3 +330,30 @@ However, a global random state is prone to modification by other code during
 execution. Thus, the only way to ensure replicability is to pass ``RandomState``
 instances everywhere and ensure that both estimators and cross-validation
 splitters have their ``random_state`` parameter set.
+
+Why do categorical variables need preprocessing in scikit-learn, compared to other tools?
+--------------------------------------------------------------------------------
+
+The problem of categorical variables is at its core a Python data science
+open issue, rather than one specific to the scikit-learn project's design.
+
+scikit-learn builds on Numpy arrays, which have no categorical data type. They
+also do not handle missing values, which is a very related problem.
```

Comment:
numpy certainly had masked arrays to handle missing values. I think you should not mention them here.

Reply:
Ok then. What is it about missing values then? Mentioning them was suggested in the Issue - I guess I didn't get why. Is it that numpy.ma are not efficient or something? 
Sorry I tried to look it up but I guess I won't find a blog on that but instead need to dive into the inner workings of scikit-learn